readthedocs/sphinx_domains/models.py

```diff
@@ -32,6 +32,11 @@ class SphinxDomain(TimeStampedModel):
         verbose_name=_('Version'),
         related_name='sphinx_domains',
     )
+    html_file = models.ForeignKey(
+        HTMLFile,
+        related_name='sphinx_domains',
+        null=True,
```

Comment:
I would also suggest `blank=True` unless we're planning to backfill this.

Reply:
The migration just makes it `null`, and it should be filled going forward. Any reason to allow `blank`?

Reply:
If we want to allow `null` on already existing objects, but force on new ones, it makes sense as it is to me.